arm/Microsoft.Authorization/policyAssignments/readme.md

```diff
@@ -39,7 +39,7 @@ With this module you can perform policy assignments across the management group,
 | `metadata` | object | `{object}` |  | The policy assignment metadata. Metadata is an open ended object and is typically a collection of key-value pairs. |
 | `nonComplianceMessage` | string | `''` |  | The messages that describe why a resource is non-compliant with the policy. |
 | `notScopes` | array | `[]` |  | The policy excluded scopes. |
-| `parameters` | _[parameters](parameters/readme.md)_ object | `{object}` |  | Parameters for the policy assignment if needed. |
+| `parameters` | object | `{object}` |  | Parameters for the policy assignment if needed. |
```

Comment:
Just to double check if that's intended

Reply:
It is. I can only suspect that there was a time when this module had a 'parameters' instead of a '.parameters' folder that caused the script to think this is a child module